.github/workflows/ci.generate.ts

```diff
@@ -27,19 +27,21 @@ const Runners = (() => {
 const prCacheKeyPrefix =
```

Comment:
Let's bump the cache key anyway on line 8 because it becomes essentially useless with a rust upgrade and will cause some needless data to be downloaded.

Reply:
Good call. Done.